frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -626,6 +850,16 @@ export const ClusterSettingsPage: React.SFC<ClusterSettingsPageProps> = ({ match
   const resources = [
     { kind: clusterVersionReference, name: 'version', isList: false, prop: 'obj' },
     { kind: clusterAutoscalerReference, isList: true, prop: 'autoscalers', optional: true },
+    {
+      kind: referenceForModel(MachineConfigPoolModel),
+      isList: true,
+      prop: 'machineConfigPools',
+    },
+    {
+      kind: referenceForModel(ClusterOperatorModel),
+      isList: true,
+      prop: 'clusterOperators',
+    },
```

Comment:
Move these watches into the `UpdateProgress` component? Then we only watch when an upgrade is in progress. (Consider the `useK8sWatch` hook instead of `Firehose`.)

Reply:
I think we want to keep `clusterOperators` here as `<UpdateStatus>` is now using it as well, but `machineConfigPools` can certainly move to `<UpdateProgress>`.  Good catch!  It prompted me to notice I was plumbing `machineConfigPools` through `<UpdateStatus>` and `<UpdatingMessage>` but not actually using it (orphaned from previous revisions).

Reply:
I think I've addressed this.  PTAL, @spadgett.